### PR TITLE
Tableau.Desktop version 22.3.0

### PR DESCRIPTION
**Pull request by s-crypt**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/87105)

### CODE REVIEW

**Comment by s-crypt** [comment]
https://github.com/microsoft/winget-pkgs/pull/87104#issuecomment-1299589981
Same issue, same installer

File: manifests/t/Tableau/Desktop/22.3.0/Tableau.Desktop.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Tableau.Desktop
+PackageVersion: 22.3.0
```
Comment (ItzLevvie) [inline manifests/t/Tableau/Desktop/22.3.0/Tableau.Desktop.installer.yaml:5]:
```suggestion
PackageVersion: 22.3.1408
```

File: manifests/t/Tableau/Desktop/22.3.0/Tableau.Desktop.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Tableau.Desktop
+PackageVersion: 22.3.0
```
Comment (ItzLevvie) [inline manifests/t/Tableau/Desktop/22.3.0/Tableau.Desktop.yaml:5]:
```suggestion
PackageVersion: 22.3.1408
```

File: manifests/t/Tableau/Desktop/22.3.0/Tableau.Desktop.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+InstallerSwitches:
+  Custom: ACCEPTEULA=1
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayName: Tableau 2022.1 (20221.22.0324.2209)
+  Publisher: Tableau Software, LLC
+  ProductCode: '{0d46353a-e530-4501-8f79-334bf28b1cc9}'
+  InstallerType: burn
+Installers:
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/t/Tableau/Desktop/22.3.0/Tableau.Desktop.installer.yaml:23]:
```suggestion
- Architecture: x64
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Replaced by PR 87982 to implement suggestions;

**Pull request closed without merging**